docs/developer/settings.rst

```diff
@@ -272,6 +272,65 @@ LFS_SEND_ORDER_MAIL_ON_PAYMENT
     If true, an e-mail is send to the customer after the customer successfully
     pays for an order
 
+
+.. _settings_sitemaps:
+
+Sitemaps
+========
+
+LFS_SITEMAPS
+    A dictionary which defines several settings for sitemaps. There are four
+    sitemaps: ``category``, ``page``, ``product`` and ``shop``. Each of them can
+    have four different settings: ``sitemap``, ``changefreq``, ``priority`` and
+    ``protocol``. All sitemaps and settings are optional. If not given the
+    default settings are used, which are::
+
+     LFS_SITEMAPS = {
+         "product": {
+             "sitemap": "lfs.core.sitemap.ProductSitemap",
+             "changefreq": "weekly",
+             "priority": 0.5,
+             "protocol": "http",
+         },
+         "category": {
+             "sitemap": "lfs.core.sitemap.CategorySitemap",
+             "changefreq": "weekly",
+             "priority": 0.5,
+             "protocol": "http",
+         },
+         "page": {
+             "sitemap": "lfs.core.sitemap.PagetSitemap",
```

Comment:
Should be **lfs.core.sitemaps.PageSitemap** not **lfs.core.sitemap.PagetSitemap**  (note Paget instead of Page)